lib/clowne/cloner.rb

```diff
@@ -59,7 +61,11 @@ def call(object, **options)
 
         plan = Clowne::Planner.enhance(plan, Proc.new) if block_given?
 
-        adapter.clone(object, plan, params: options)
+        adapter.clone(object, plan, params: options, only: only)
```

Comment:
I don't really prefer idea with additional `only` key and processing it within a `inject cycle`...How do you think it is possible to filter out (or maybe generate new with option `only`) our plan at this stage and
1. Get ride of `only` key
2. Don't touch the logic of `Adaptes::Base.clone`

?

Also I don't see how you pass this `only` key to nested cloners.

Reply:
Yeah, make sense. What about:

```ruby
plan.filter_declarations(only) unless only.nil?
```

?

Reply:
Sounds good! 👍 
Except that you can "hide" `nil?` condition inside the method
```
def filter_declarations(only)
   return self if only.nil?
end

# and get a clean call ^^
plan.filter_declarations(only)
```